sqldelight-idea-plugin/src/test/kotlin/com/squareup/sqldelight/intellij/GoToDeclarationHandlerTest.kt

```diff
@@ -0,0 +1,29 @@
+package com.squareup.sqldelight.intellij
+
+import com.alecstrong.sqlite.psi.core.psi.SqliteIdentifier
+import com.google.common.truth.Truth.assertThat
+import com.intellij.project.rootManager
+import org.jetbrains.kotlin.psi.psiUtil.getStrictParentOfType
+
+class GoToDeclarationHandlerTest : SqlDelightProjectTestCase() {
+  private val goToDeclarationHandler = SqlDelightGotoDeclarationHandler()
+
+  fun testMethodGoesToIdentifier() {
+    val tempRoot = myModule.rootManager.contentRoots.single()
+    myFixture.openFileInEditor(
+        tempRoot.findFileByRelativePath("src/main/kotlin/com/example/SampleClass.java")!!
+    )
+    var offset = file.text.indexOf("someQuery")
+    val sourceElement = file.findElementAt(offset)
+
+    val elements = goToDeclarationHandler.getGotoDeclarationTargets(sourceElement, offset, editor)
```

Comment:
Is there a way to invoke this on `myFixture` to avoid interacting with the handler directly?

Reply:
yea, I tried `myFixture.runAction(IdeActions.GO_TO_DECLARATION)` but the implementation of `IdeFocusManager` the headless environment uses cant move focus around. I tried writing my own but ran into more dead ends.